.gitattributes

```diff
@@ -1,2 +1,2 @@
 numba_scipy/_version.py export-subst
-numba_scipy/special/signatures.py binary
+# numba_scipy/special/signatures.py binary
```

Comment:
If this is dead code, I think it should go? Not even sure why it was being treated as binary. Maybe so that Windows EOL's can pass unchanged?

Reply:
Agh, that was a local change, but I couldn't get a proper diff without it.